### DISCUSSION TITLE
Issue with validation of data quality in service metadata

### DISCUSSION THREAD

**Discussion by thomruhl** [INSPIRE-MIF/helpdesk-validator#934] [Q&A] [answered]
Hello,

There seems to be an issue with the "Data Quality" section of "Conformance Class 3" results of the validator when checking a network service metadata.

In our DQ_ConformanceResults, a wrong reference to Commission Regulation 1089/2010 instead of 976/2009 doesn't get flagged by the validator like we would expect it to be.

In attachment, you'll find the report.

Best regards,
Thomas Ruhl

[Test run on 11_13 - 21.12.2022 with test suite Conformance Class 4_ INSPIRE Network Services metadata.pdf](https://github.com/INSPIRE-MIF/helpdesk-validator/files/10285525/Test.run.on.11_13.-.21.12.2022.with.test.suite.Conformance.Class.4_.INSPIRE.Network.Services.metadata.pdf)

**Comment by fabiovinci** [comment]
Dear @thomruhl,

could you please send us the metadata to check the encoding? Thanks

**Comment by thomruhl** [comment]
Hi @fabiovinci,

Here is the metadata in attachment.

Best,
Thomas

[beb71088-7403-48dc-9a8d-3d00d2e70dff.zip](https://github.com/INSPIRE-MIF/helpdesk-validator/files/10426103/beb71088-7403-48dc-9a8d-3d00d2e70dff.zip)

**Comment by fabiovinci** [comment, marked as answer]
Dear @thomruhl,

thank you for the metadata.

The check you are referring to "should" be in the Conformance Class 4 (the "Data Quality" section of "Conformance Class 3" refers to another metadata element) but it is only a "Recommendation" (see [section 4.2.2.1. Conformity](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#4221-conformity) of the MD TG) and they are not implemented.

![image](https://user-images.githubusercontent.com/22212406/213175041-81692f15-f4db-4717-bfa1-c8944ce35f45.png)

I hope that this helps to clarify.
